blockprotocol/blockprotocol
# Bump Type Editor version

## 🌟 What is the purpose of this PR?

This updates the Type Editor to a version that implements the changes required by #1041 


## ❓ How to test this?

- Ensure the type editor works as expected
- Check the generated type doesn't have `additionalProperties` within it

## 📹 Demo

<img width="735" alt="image" src="https://user-images.githubusercontent.com/25749103/221375833-031de400-346b-4773-bb1c-34edcaf47514.png">
